integration_tests/pages/checkYourBooking.ts

```diff
@@ -33,5 +33,14 @@ export default class CheckYourBookingPage extends Page {
 
   changeRequestMethod = (): PageElement => cy.get('[data-test="change-request-method"]')
 
-  bookButton = (): PageElement => cy.get('[data-test=submit]')
+  submitBooking = (): void => {
```

Comment:
Complexity here is because just doing a `.click()` and then checking the button has been disabled won't work because `click()` won't resolve until after the next page has loaded.